packages/sdk/src/modules/FusePools.ts

```diff
@@ -18,16 +18,16 @@ export type LensPoolsWithData = [
 
 export function withFusePools<TBase extends MidasBaseConstructor>(Base: TBase) {
   return class FusePools extends Base {
-    async fetchFusePoolData(poolId: string, address?: string): Promise<FusePoolData> {
+    async fetchFusePoolData(poolId: string, overrides?: CallOverrides): Promise<FusePoolData> {
```

Comment:
we'll have to be careful here with our dependent libs. The bots for instance call this function a lot, so might have to check if their usage is affected

Reply:
Actually, looks like you've already made the necessary changes in the liquidations module, so we're good to go!